src/components/File/File.js

```diff
@@ -0,0 +1,190 @@
+/* ./src/components/{Component}/{Component}.js */
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import { Icon, MediaObject } from '../..';
+
+const RATIOS = [
+  '1-by-1',
+  '4-by-3',
+  '16-by-9'
+];
+
+export const File = (props) => {
+  const {
+    actions,
+    className,
+    externalIcon,
+    fileType,
+    hideTitle,
+    image,
+    isLoading,
+    ratio,
+    title,
+    ...rest
```

Comment:
Although the linter allows this, can you add a comma to be consistent.

Reply:
no: https://github.com/eslint/eslint/issues/3627